Makefile

```diff
@@ -4,7 +4,7 @@ REGISTRY ?= envoyproxy
 IMAGE := $(REGISTRY)/$(PROJECT)
 INTEGRATION_IMAGE := $(REGISTRY)/$(PROJECT)_integration
 MODULE = github.com/envoyproxy/ratelimit
-GIT_REF = $(shell git describe --tags || git rev-parse --short=8 --verify HEAD)
+GIT_REF = $(shell git rev-parse --verify HEAD)
```

Comment:
Should we retain the tag on release builds if we ever do releases for some reason?

Reply:
Yup you are right, I put that back with a tweak to only grab the version tag if it's on a tagged release.